src/main/resources/hudson/plugins/batch_task/BatchTaskInvoker/Config/help-project.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    The <strong>Full project name</strong>
+
+    <p>
+        For jobs inside a folder it is always a path like <strong>foo/bar</strong>
```

Comment:
NIT: maybe makes sense to reference other use-cases: E.g. multibranch projects, etc.